bluesky/sim/qtgl/mainmanager.py

```diff
@@ -14,7 +14,7 @@
 # Local imports
 from bluesky import settings
 from .simevents import SimStateEventType, SimQuitEventType, BatchEventType, \
-    BatchEvent, StackTextEvent, SimQuitEvent, SetNodeIdType, \
+    BatchEvent, StackTextEvent, StackTextEventType, SimQuitEvent, SetNodeIdType, \
```

Comment:
This is universally problematic at the moment - need to 'pep' the whole source tree.